actions/icon_overlay.rb

```diff
@@ -72,15 +63,15 @@ def self.source_icons_directory
         source_path = ENV['ICON_OVERLAY_SOURCE_PATH']
         if source_path
           source_path = File.expand_path(source_path)
-        else
-          source_path = File.expand_path(File.join(ENV['ICON_OVERLAY_ASSETS_BUNDLE'], 'AppIcon.appiconset')) if ENV['ICON_OVERLAY_ASSETS_BUNDLE']
+        elsif ENV['ICON_OVERLAY_ASSETS_BUNDLE']
+          source_path = File.expand_path(File.join(ENV['ICON_OVERLAY_ASSETS_BUNDLE'], 'AppIcon.appiconset'))
         end
         source_path
       end
 
       def self.destination_icons_directory
         destination_path = ENV['ICON_OVERLAY_ASSETS_BUNDLE']
-        if destination_path
+        if destination_path # rubocop:disable Style/ConditionalAssignment
```

Comment:
Same as above: 
```
destination_path = if !ENV['ICON_OVERLAY_ASSETS_BUNDLE'].empty?
                      File.expand_path(File.join(destination_path, 'AppIcon.appiconset'))
                   else
                      File.expand_path(ENV['ICON_OVERLAY_SOURCE_PATH'])
                   end
```

Reply:
And can get rid of `destination_path` afterward

Reply:
As above?